smoke_tests/happypath/resume.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Resume to continue existing application
+
+  Background:
```

Comment:
Can you give the background a description for what it's doing
Something like: 'Create saved case'